.travis.yml

```diff
@@ -27,6 +27,8 @@ env:
     - OPTS="--test=artik10" INSTALL_TIZEN_DEPS=yes
     - OPTS="--test=artik053" INSTALL_TIZENRT_DEPS=yes
     - OPTS="--test=misc"
+    - OPTS="--test=nosnapshot"
+    - OPTS="--test=minimalprofile"
```

Comment:
I'd write it with dashes to be similar to the build options (`no-snapshot` and `minimal-profile`).